src/native/managed/cdacreader/src/Contracts/RuntimeTypeSystem_1.cs

```diff
@@ -0,0 +1,157 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts;
+
+
+
+internal partial struct RuntimeTypeSystem_1 : IRuntimeTypeSystem
+{
+    private readonly Target _target;
+    private readonly TargetPointer _freeObjectMethodTablePointer;
+
+    // TODO(cdac): we mutate this dictionary - copies of the Metadata_1 struct share this instance.
```

Comment:
Support for invalidating our view of memory and re-reading to fill in the world should be a pretty soon now project, but I'm ok with having checking in the current state here, and fixing it later when we add that support.